modules/shredder/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/shredder/batch/Cloud.scala

```diff
@@ -66,4 +90,54 @@ object Cloud {
     val (bucket, s3Key) = S3.splitS3Key(key)
     client.doesObjectExist(bucket, s3Key)
   }
+
+  def sendToSns(client: AmazonSNS, topic: String, messageGroupId: String, message: String): Either[Throwable, Unit] =
+    Either.catchNonFatal {
+      val request = new PublishRequest()
+        .withTargetArn(topic)
+        .withMessage(message)
+        .withMessageGroupId(messageGroupId)
+      client.publish(request)
```

Comment:
What does `publish()` return ? Don't we want to read it?

Reply:
These results are mostly contain things like `messageId`, `sequenceNumber`, `mD5OfMessageBody` etc. Since we don't use them, I just ignored them as it was done previously with SQS and S3.